src/workers/atlas_worker.js

```diff
@@ -64,23 +82,29 @@ export default class AtlasWorker extends PeriodicWorker {
   async tryWithChallenge(challenge) {
     try {
       if (this.failedChallengesCache.didChallengeFailRecently(challenge.challengeId)) {
+        this.atlasChallengeMetrics.inc({status: atlasChallengeStatus.failedRecently});
```

Comment:
I'm not sure if we should track this as this is only a result of the `failed` metric and this will generate a lot of noise.

Reply:
I'd rather collect as many metrics as possible initially to see what the node is exactly doing.